avm/res/app/container-app/README.md

```diff
@@ -0,0 +1,1047 @@
+# Container Apps `[Microsoft.App/containerApps]`
+
+This module deploys a Container App.
+
+## Navigation
```

Comment:
Please update the version of your fork as there have been some updates to the ReadMe utility (like a disclaimer for data collection being added).

Reply:
I have updated my branch with upstream main and rerun Set-AVMModule.